opendbc/safety/safety/safety_rivian.h

```diff
@@ -0,0 +1,161 @@
+#pragma once
+
+#include "safety_declarations.h"
+
+static bool rivian_longitudinal = false;
+
+static void rivian_rx_hook(const CANPacket_t *to_push) {
+  int bus = GET_BUS(to_push);
+  int addr = GET_ADDR(to_push);
+
+  if (bus == 0)  {
+    // Vehicle speed
+    if(addr == 0x208){
+      float speed =  (((GET_BYTE(to_push, 6) << 8) | GET_BYTE(to_push, 7)) * 0.01);
+      vehicle_moving = speed != 0;
+      UPDATE_VEHICLE_SPEED(speed / 3.6);
+    }
+
+    // Driver torque
+    if (addr == 0x380) {
+      int torque_driver_new = ((((GET_BYTE(to_push, 2) << 4) | (GET_BYTE(to_push, 3) >> 4)) * 0.1) - 205);
+      update_sample(&torque_driver, torque_driver_new);
```

Comment:
`driver_torque_allowance` is an integer, but you're converting this to a float (that's getting truncated to an int with update_sample), are you sure this is doing anything?

Reply:
so subtract `205/0.1=2050` here